### DISCUSSION TITLE
Should the vscode extension be moved into its own repository?

### DISCUSSION THREAD

**Discussion by SeniorMars** [nvarner/typst-lsp#18] [General]
I just wanted a discussion around this.

I feel language servers should be separated from editors and treated only as clients, but I understand that the LSP spec came from vscode.

**Comment by beeb** [comment]
I don't think so, rust-analyzer also has both in the same repo. Besides, monorepos are trendy these days :)

**Reply by CGMossa** [reply]
I'd follow RA's example.. But I don't know anything about this.
